scripts/ci-build-kubernetes.sh

```diff
@@ -135,26 +135,24 @@ main() {
 # can_reuse_artifacts returns true if there exists Kubernetes artifacts built from a PR that we can reuse
 can_reuse_artifacts() {
     for IMAGE_NAME in "${IMAGES[@]}"; do
-        if ! docker pull "${REGISTRY}/${IMAGE_NAME}:${KUBE_IMAGE_TAG}"; then
-            echo "false" && return
+        if ! docker manifest inspect "${REGISTRY}/${IMAGE_NAME}:${KUBE_IMAGE_TAG}" >/dev/null; then
```

Comment:
`docker manifest inspect` is way faster than `docker pull` if all we need to do is check if the image exists.